src/RequestForm.js

```diff
@@ -1046,19 +1051,19 @@ class RequestForm extends React.Component {
               viewUserPath={() => this.onViewUserPath(selectedUser, patronGroup)}
               patronBlocks={patronBlocks || []}
             />
-            {showDeclaredLostError &&
+            {isErrorModalOpen &&
               <ErrorModal
-                id="declared-lost-modal"
-                data-test-declared-lost-modal
```

Comment:
`data-test` attribute has been removed due to not being used anywhere (at least for now). For tests, the `data-test-error-modal` attribute built into ErrorModal is used.